tests/helpers/module-for-acceptance.js

```diff
@@ -1,23 +0,0 @@
-import { module } from 'qunit';
```

Comment:
i didn't see much value in keeping this file around given the nature of tests written in this project